### PR TITLE
Fix script for requesting phone number on staging / production

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

Need to tweak the `request_long_codes.sh` script because on staging and production we have custom keywords for the pools. :/

Fixing this simplifies the code substantially though 🎉 

Also cleaned up the drain script slightly and incorporated some AI suggestions! 🤖 

## Related Issues | Cartes liées

* https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/396

## Before merging this PR

Read code suggestions left by the
[cds-ai-codereviewer](https://github.com/cds-snc/cds-ai-codereviewer/) bot. Address
valid suggestions and shortly write down reasons to not address others. To help
with the classification of the comments, please use these reactions on each of the
comments made by the AI review:

| Classification      | Reaction | Emoticon |
|---------------------|----------|----------|
| Useful              | +1       | 👍        |
| Noisy               | eyes     | 👀        |
| Hallucination       | confused | 😕        |
| Wrong but teachable | rocket   | 🚀        |
| Wrong and incorrect | -1       | 👎        |

The classifications will be extracted and summarized into an analysis of how helpful
or not the AI code review really is.

## Test instructions | Instructions pour tester la modification

Add some numbers to staging and production.
(Note: currently we are at the limit of numbers on staging so the script will fail. Delete a few and then try)

## Release Instructions | Instructions pour le déploiement

None.

## Reviewer checklist | Liste de vérification du réviseur

* [ ] This PR does not break existing functionality.
* [ ] This PR does not violate GCNotify's privacy policies.
* [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
* [ ] This PR does not significantly alter performance.
* [ ] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: scripts/manage_longcodes/drain_pool.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ printf "                         WARNING!!!!\n"
 printf "  This will delete all but one long code from a Pinpoint pool!\n"
 printf "         You do not want to run this or production!\n"
 printf "\n------------------------------------------------------------\n"
-printf "Are you sure you want to continue?"
+printf "Are you sure you want to continue?\n"
 echo -n "If so, type 'drain'> "
```
[the remark this answers is not in the document]
Comment (sastels) [reply scripts/manage_longcodes/drain_pool.sh:21]: yes this is definitely where the AI review is useful!

Slightly odd that it didn't flag the same code in the other file

File: scripts/manage_longcodes/drain_pool.sh [hunk 2]
```diff
@@ -26,15 +26,15 @@ if [ "$check" != "drain" ]; then
     exit 1
 fi
 
-if aws pinpoint-sms-voice-v2 describe-pools --pool-ids $1; then    
+if aws pinpoint-sms-voice-v2 describe-pools --pool-ids $1 > /dev/null 2>&1; then
```
[the remark this answers is not in the document]
Comment (sastels) [reply scripts/manage_longcodes/drain_pool.sh:29]: I thought of that but figured 🤷  but tbh I just want to hide the output when it's successful, so we can do that (but not hide the errors) by getting rid of the `2>&1`.

File: scripts/manage_longcodes/request_long_codes.sh [hunk 3]
```diff
@@ -19,20 +19,16 @@ if [ -z "$2" ]; then
...
 poolId=$2
 
-
 printf "\n------------------------------------------------------------\n"
 printf "                        WARNING!!!!\n"
 printf "  This will add new phone numbers to a Pinpoint pool\n"
 printf "    You might not want to run this in production!\n"
 printf "\n------------------------------------------------------------\n"
-printf "Are you sure you want to continue?"
+printf "Are you sure you want to continue?\n"
```
[the remark this answers is not in the document]
Comment (sastels) [reply scripts/manage_longcodes/request_long_codes.sh:31]: It is consistent...

File: scripts/manage_longcodes/request_long_codes.sh [hunk 4]
```diff
@@ -41,24 +37,17 @@ if [ "$check" != "request" ]; then
...
-        numberStatus=$(aws pinpoint-sms-voice-v2 describe-phone-numbers \
-          --phone-number-ids $number \
-          | jq '.PhoneNumbers[0].Status')
-    done
-
-    aws pinpoint-sms-voice-v2 associate-origination-identity \
-      --pool-id $poolId \
-      --origination-identity $number \
-      --iso-country-code CA
+    echo "Requested $number"
```
[the remark this answers is not in the document]
Comment (sastels) [reply scripts/manage_longcodes/request_long_codes.sh:52]: yeah that's a good idea.

**Review by ben851 — APPROVED** [review]
LGTM

**Pull request merged**